### ISSUE TITLE
test KubeRay candidate v1.0.0-rc.0

### ISSUE THREAD

**Issue by jbusche** [opendatahub-io/distributed-workloads#135]
A new version of KubeRay Operator is out.  https://github.com/ray-project/kuberay/tree/v1.0.0-rc.0/
Which needs testing...

**Comment by jbusche** [comment]
I've created a branch to test it:
https://github.com/jbusche/distributed-workloads/tree/kuberay-100-rc.0

and have applied this kfdef in my cluster for testing:
```
apiVersion: kfdef.apps.kubeflow.org/v1
kind: KfDef
metadata:
  name: codeflare-stack
  namespace: opendatahub
spec:
  applications:
  # CodeFlare
  - kustomizeConfig:
      repoRef:
        name: manifests
        path: codeflare-stack
    name: codeflare-stack
  # KubeRay
  - kustomizeConfig:
      repoRef:
        name: manifests
        path: ray/operator
    name: ray-operator
  repos:
    # ODH Core component manifests
  - name: manifests
    #uri: https://github.com/opendatahub-io/distributed-workloads/tarball/main
    uri: https://github.com/jbusche/distributed-workloads/tarball/kuberay-100-rc.0
```

When applied, the operator looks good:
```
oc describe pod kuberay-operator-7d768cd4b8-9gvnc |grep Image:
    Image:         quay.io/kuberay/operator:v1.0.0-rc.0
```

**Comment by jbusche** [comment]
Tried the guided demos
0_basic_ray.ipynb
2_basic_jobs.ipynb
and they look good using the Ray.

**Comment by jbusche** [comment]
Did a TwistLock Vulnerability scan, only 5 mediums and 1 high Vulnerability
```
severityCHML    cvss    riskFactors     cve     link    hasFix  status  packageType     packageName     packageVersion  packageLicense  packageBinaryPkgs       packagePath
M       6.2     DoS - Low,Has fix,Medium severity,Recent vulnerability  PRISMA-2023-0056        https://github.com/sirupsen/logrus/issues/1370  Y       fixed in v1.9.3 go      github.com/sirupsen/logrus      v1.8.1                  /manager
M       5.3     Attack complexity: low,Attack vector: network,DoS - Low,Has fix,Medium severity,Recent vulnerability    CVE-2023-29409  https://nvd.nist.gov/vuln/detail/CVE-2023-29409 Y       fixed in 1.20.7, 1.19.12        app     go      1.19.10                 /manager
M       6.1     Attack complexity: low,Attack vector: network,Has fix,Medium severity,Recent vulnerability      CVE-2023-39319  https://nvd.nist.gov/vuln/detail/CVE-2023-39319 Y       fixed in 1.21.1, 1.20.8 app     go      1.19.10                 /manager
M       6.5     Attack complexity: low,Attack vector: network,Has fix,Medium severity,Recent vulnerability      CVE-2023-29406  https://nvd.nist.gov/vuln/detail/CVE-2023-29406 Y       fixed in 1.20.6, 1.19.11        app     go      1.19.10                 /manager
M       6.1     Attack complexity: low,Attack vector: network,Has fix,Medium severity,Recent vulnerability      CVE-2023-39318  https://nvd.nist.gov/vuln/detail/CVE-2023-39318 Y       fixed in 1.21.1, 1.20.8 app     go      1.19.10                 /manager
H       7.5     Attack complexity: low,Attack vector: network,DoS - High,Has fix,High severity,Recent vulnerability     CVE-2023-39533  https://nvd.nist.gov/vuln/detail/CVE-2023-39533 Y       fixed in 1.20.7, 1.19.12        app     go      1.19.10                 /manager
```

**Comment by jbusche** [comment]
This was successful - marking it as done and will do the real 1.0.0 release next.
